src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,17 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import { User } from '../../types/User';
+
+interface UserInfoProps {
+  user: User | null
+}
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
+  if (user === null) {
+    return null;
+  }
+
+  return (
+    <a className="UserInfo" href={`mailto:${user.email}`}>
+      {user.name}
+    </a>
+  );
```

Comment:
You can just use conditional rendering
https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator
```suggestion
  return (
    {user && (
      <a className="UserInfo" href={`mailto:${user.email}`}>
      {user.name}
      </a>
    )}
  );
```